config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en/ do
+    get "users/show"
+    get "users/new"
```

Comment:
you can remote line number 3 and 4

Reply:
I'm sorry, can you please explain more, I don't understand what I should do here